src/status_im2/subs/shell.cljs

```diff
@@ -135,9 +148,18 @@
  :shell/private-group-chat-card
  (fn [[_ id] _]
    [(re-frame/subscribe [:chats/chat id])
-    (re-frame/subscribe [:communities])])
- (fn [[chat communities] [_ id]]
-   (private-group-chat-card chat id communities)))
+    (re-frame/subscribe [:communities])
+    (re-frame/subscribe [:contacts/contacts])
+    (re-frame/subscribe [:multiaccount])])
+ (fn [[chat communities contacts current-multiaccount] [_ id]]
+   (let [from         (get-in chat [:last-message :from])
+         contact      (when from (multiaccounts/contact-by-identity contacts from))
```

Comment:
Why do we need if from exists?
Is it possible to have a message without a sender, Is that a case for system messages maybe?